src/pages/Dashboard/Dashboard.js

```diff
@@ -13,6 +14,11 @@ function Dashboard() {
 
     const { allFields, currentField } = useSearchSelector();
     const { setInput, setSelected } = useSearchDispatch();
+    const { loadDesignations } = useBooksDispatch();
+
+    useEffect(() => {
+        loadDesignations();
+    }, []);
```

Comment:
I was wondering where the designations loading has went from the main App component

Reply:
the designations loads now from the Dashboard component after successfully logging in , if the user logs in and then logs out , the app component doesnt render again and the designations wont be loaded